src/bindings/python/setup.py

```diff
@@ -54,6 +54,7 @@
     "openvino.runtime.opset7",
```

Comment:
better to drop this file at all. It's not a production code, used somewhere in CI only.

Reply:
yes, we have the task for it -- 81848